.github/workflows/update-tailwind.yml

```diff
@@ -21,7 +21,7 @@ jobs:
       run: |
         git config --local user.email "github-actions@example.com"
         git config --local user.name "GitHub Actions"
-        git commit -am "Update tailwind.css" || echo "No changes to commit"
+        git diff --quiet app/assets || git commit -am "Update tailwind.css"
```

Comment:
This uses `||` (not `&&`) because [`git diff --quiet`](https://git-scm.com/docs/git-diff#Documentation/git-diff.txt---quiet) returns a truthy exit code when there are no differences.  (In shell scripting, an exit code of `0` is truthy and `1` is falsy.)